science/iverilog/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        steveicarus iverilog 11_0 v
+github.setup        steveicarus iverilog 12_0 v
```

Comment:
please include "github.tarball_from archive" or preferably "releases" if upstream provides those as they might already contain the "configure" file.

Reply:
that should have been done now - everything can be done later but then never happens.... 😉

Reply:
The perfect is the enemy of the better.

Reply:
They don't, but I'll add it in a new PR.